src/track/track.cpp

```diff
@@ -323,6 +324,23 @@ mixxx::ReplayGain Track::getReplayGain() const {
     return m_record.getMetadata().getTrackInfo().getReplayGain();
 }
 
+bool Track::trySetReplayGainFromString(const QString& gainStr) {
```

Comment:
This is the wrong place for parsing. The `Track` object should not be responsible for validating user input.

Reply:
done